src/pymovements/datasets/toy_dataset_eyelink.py

```diff
@@ -182,6 +188,10 @@ class ToyDatasetEyeLink(DatasetDefinition):
         },
     )
 
+    trial_columns: list[str] = field(
+        default_factory=lambda: ['task', 'trial_id', 'screen_id', 'point_id'],
```

Comment:
iirc `point_id` is not a trial but part of of judo sequence? @prassepaul

Reply:
good catch. probably both screen_id and point_id are not actual trial columns. just the task and the trial id

Reply:
unsure -- `screen_id` might be from the reading trials, but probably is covered already with `trial_id`. @jakobchwastek knows best

Reply:
The reading task consists of a just a single text (the lion and the hare as far as I remember).
The multiple screens of the text should be assigned to the same trial.

To further clarify, `task` is not given in the file itself, but gained from the patterns passed to `from_asc()`:

https://github.com/aeye-lab/pymovements/blob/913c372f83d1d19eada650f28fd2ec52d741f0ad/src/pymovements/datasets/toy_dataset_eyelink.py#L193-L211

Reply:
I removed `point_id` and `screen_id` from the trials

Reply:
many researchers treat one screen as one trial but should then probably be easy for them to split across screens if they want to. e.g. sb-sat did it in their paper, and hence we adopted it for our papers and the [datasets](https://github.com/aeye-lab/pymovements/blob/d5c4cafd72dfe02cd7feda14b7a7f213b53bf715/src/pymovements/datasets/sb_sat.py#L197-L202) but it can be up to the researcher. I think have finer-grained trials is generally better but it does not really matter for the toy dataset

Reply:
yes, i think we would make sure that we define and document these things clearly and than first take care of being consistent within pymovements. We can then further support different workflows regarding special needs